compiler/runtime/CodeCacheTypes.cpp

```diff
@@ -19,7 +19,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
  *******************************************************************************/
 
-#include "CodeCacheTypes.hpp"
+#include "runtime/CodeCacheTypes.hpp"
```

Comment:
We don't have to do this here as we are inconsistent elsewhere but what do you think about alphabetizing the include directives in the compiler subdirectories?

Reply:
not an unreasonable goal to move towards, I think